ale/base/type_sensor.py

```diff
@@ -112,3 +113,72 @@ def ephemeris_stop_time(self):
           Ephemeris stop time for an image
         """
         return self.ephemeris_start_time + self.exposure_duration
+
+class Radar():
+    @property
+    def name_model(self):
+        """
+        Returns Key used to define the sensor type. Primarily
+        used for generating camera models.
+
+        Returns
+        -------
+        : str
+          USGS SAR (synthetic aperture radar) model
+        """
+        return "USGS_ASTRO_SAR_MODEL"
+
+    @property
+    def ephemeris_time(self):
+        newstart = spice.str2et(str(self.utc_start_time))
+        newend = spice.str2et(str(self.utc_stop_time))
+        return [newstart, newend]
+
+    @property
+    def wavelength(self):
+        """
+        Returns the wavelength used for image acquistion. 
+
+        Returns
+        -------
+        : double
+          Wavelength used to create an image in meters
+        """
+        return self.wavelength
+
+    @property
+    def line_exposure_duration(self):
+        """
+        Returns the exposure duration for each line.
+
+        Returns
+        -------
+        : double
+          Exposure duration for a line
+        """
+        return self.line_exposure_duration
+
+    @property
+    def scaled_pixel_width(self):
+        """
+        Returns the pixel width scaled by #FIXME
+
+        Returns
+        -------
+        : double
+          Scaled pixel width
+        """
+        return self.scaled_pixel_width
+
+    @property
+    def range_conversion_coefficients(self):
+        """
+        Returns the range conversion coefficients
+
+        Returns
+        -------
+        : list
+          Coefficients needed for range conversion
+        """
+        return self.range_conversion_coefficients
```

Comment:
All of these methods are cyclic and will end up just recursively calling themselves until they hit the max depth.